images/win/scripts/Tests/BizTalk.Tests.ps1

```diff
@@ -0,0 +1,19 @@
+################################################################################
+##  File:  BizTalk.Tests.ps1
+##  Desc:  Test BizTalk project build component installed.
+################################################################################
+
+Describe "BizTalk Build Component Setup" {
```

Comment:
```
Describe "BizTalk Build Component Setup" -Skip:(Test-IsWin16)  {
```

To avoid running this test on Windows16

Reply:
This is only invoked in Install-BizTalkBuildComponent.ps1, which is only in windows2019.json, I suppose no need to add this skip condition? please confirm.

Reply:
We run every test twice:
1) Right after tool installation (`Invoke-PesterTests -TestFile "BizTalk" -TestName "BizTalk Build" `)
2) At the end of the build
At the end of the build we run all tests so condition is necessary

Reply:
Thanks Maxim for your confirmation, updated.